libs/nx-remotecache-s3/package.json

```diff
@@ -30,15 +30,15 @@
   },
   "homepage": "https://github.com/robinpellegrims/pellegrims/tree/master/libs/nx-remotecache-s3",
   "peerDependencies": {
-    "nx": ">=17.0.0"
+    "nx": ">=19.0.0"
```

Comment:
This was lagging behind a version, I think it would make sense it goes up to 19 as well?